ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/visualization-selector/visualization-selector.view.js

```diff
@@ -0,0 +1,226 @@
+/**
```

Comment:
✏️ Would be nice to convert this file to typescript as well

Reply:
✏️ Probably wouldn't hurt to remove 'view' from the name as well. I think 'view' implies a marionette component.